content/departments/bizops/analytics/faqs.md

```diff
@@ -0,0 +1,33 @@
+# Data & Analytics FAQs
+
+Below are answers to common questions teammates at Sourcegraph have.
+
+### Current seat limit, ARR, company type, and more are outdated on the [Server Instances Overview](https://sourcegraph.looker.com/dashboards/167). How can we get this updated?
```

Comment:
Joe pulled the current_seat_limit field from salesforce so it's actually just not an option anymore. May want to remove it from this line to avoid confusion that we can currently access that information in Looker. I'll add to the source of truth doc the places you can find license seat count.